src/TestExplorer/TestRunner.ts

```diff
@@ -584,16 +585,37 @@ class TestRunState implements iTestRunState {
             this.testRun.failed(this.testItems[index], new vscode.TestMessage(message));
         }
         this.testItems.splice(index, 1);
+        this.lastTestItem = this.currentTestItem;
         this.currentTestItem = undefined;
     }
 
     // set test item to have been skipped
     skipped(index: number): void {
         this.testRun.skipped(this.testItems[index]);
         this.testItems.splice(index, 1);
+        this.lastTestItem = this.currentTestItem;
         this.currentTestItem = undefined;
     }
 
+    // started suite
+    startedSuite() {
+        //
```

Comment:
If there's nothing to do here we should update the comment
```suggestion
        // Nothing to do here
```